### ISSUE TITLE
Route snapper behaviour around footpaths and elevated footpaths

### ISSUE THREAD

**Issue by Sparrow0hawk** [acteng/atip#176]
Observed a user doing this during testing session. It wasn't super clear to the user why this happened but on actually looking at the map it makes sense (two footpaths cross one over the other).

# Steps to reproduce

1. Navigate to https://acteng.github.io/atip/scheme.html?authority=York#17.76/53.95755/-1.09187
2. Try and draw a footpath route from the footpath crossing station road walk up the city wall footpath in a northerly direction

## Behaviour

You get the behaviour shown because the footpath on the walls is elevated and not directly reachable on the route sketched. What we've got here is the correct behaviour I believe but it wasn't clear to a user.

Uploading atip-york-footpath.mov…

**Comment by dabreegster** [comment]
The video didn't upload (sometimes GH flakes on this); could you retry please? I got something like this, but not sure if that's the problem you mean:
![Screenshot from 2023-05-18 10-27-18](https://github.com/acteng/atip/assets/1664407/79f4f18b-04b3-4589-bd28-988760bbe8f5)

**Comment by Sparrow0hawk** [comment]
Yeah i'm retrying to little avail but what you've got there is exactly what i'm describing.

**Comment by dabreegster** [comment]
Google Maps for reference: https://www.google.com/maps/dir/53.958492,-1.0905469/Cycle+Heaven,+Station+Rd,+York+YO24+1AY/@53.9572584,-1.0921919,17.17z/data=!4m10!4m9!1m0!1m5!1m1!1s0x487930550d04e92b:0x99cd6d1701efff0a!2m2!1d-1.0925836!2d53.9576762!3e2!5i1
![Screenshot from 2023-05-18 10-31-15](https://github.com/acteng/atip/assets/1664407/115df845-b8d5-4f1e-8f32-cbd048060a1a)

**Comment by Sparrow0hawk** [comment]
> But what should we do to explain?

I am not entirely sure at this stage, something to have a think on
